cmd/prometheustosfx/main.go

```diff
@@ -176,11 +188,23 @@ func main() {
 			EnvVar: "SFX_SCRAPPER_SEND_RATE",
 			Usage:  fmt.Sprintf("Rate at which data is queried from cAdvisor and send to SignalFx. Possible values: %v", getMapKeys(dataSendRates)),
 		},
+		cli.IntFlag{
+			Name:   cadvisorPort,
+			Value:  4194,
+			EnvVar: "SFX_SCRAPPER_CADVISOR_PORT",
+			Usage:  fmt.Sprintf("Port on which cAdvisor listens."),
```

Comment:
Not sure what you've meant here. Confused with line numbers to which this comment assigned.